api/src/routes/traces.ts

```diff
@@ -35,6 +35,7 @@ app.get("/v1/traces", async (ctx) => {
   const spans = await db.query.otelSpans.findMany({
     where: sql`inner->>'scope_name' = 'fpx-tracer'`,
     orderBy: desc(sql`inner->>'end_time'`),
+    limit: 1000,
```

Comment:
this is a little left field but i finally hit the issue where i had so many spans it effectively broke the UI. i put a limit of 1000 here which seems okay but... who knows